Services/Simulations.cs

```diff
@@ -546,7 +551,8 @@ public async Task<bool> TryToSetDeviceDeletionCompleteAsync(string simulationId)
             simulation.DevicesCreationComplete = false;
             simulation.DeviceCreationJobId = null;
             simulation.DevicesCreationStarted = false;
-
+            simulation.DeleteDevicesOnce = false;
+            
```

Comment:
remove spaces

Reply:
I did not add a new space, Devis. This was existing before. In master line 549 will be blank. I am just retaining the exisiting empty line.